.github/workflows/ci.yml

```diff
@@ -150,7 +150,7 @@ jobs:
           - macOS_arm64_static
           - macOS_arm64_mixed
           - macOS_x86_64
-    runs-on: macos-latest
+    runs-on: macos-12
```

Comment:
Shouldn't be macos-11 ?
See https://github.com/kiwix/kiwix-build/issues/595#issuecomment-1514849806 (and slack discussion)

Reply:
yes, bit didn't want to change without confirmation or within this PR.